tfjs-backend-webgpu/src/kernels/conv2d_mm_vec4_webgpu.ts

```diff
@@ -113,7 +159,7 @@ export class Conv2DMMVec4Program implements WebGPUProgram {
         throw new Error('Leakyrelu is not supported.');
       } else {
         activationSnippet = `
-        vec4 activation(vec4 a, ivec4 outCoord) {
+          vec4 activation(vec4 a, ivec4 outCoord) {
```

Comment:
nit: should be indented the same amount as the corresponding `}`

Reply:
Done.